bokeh/core/validation/check.py

```diff
@@ -76,6 +79,21 @@ def silence(warning: int, silence: bool = True) -> Set[int]:
         __silencers__.remove(warning)
     return __silencers__
 
+def is_silenced(warning):
+    ''' Check if a warning has been silenced.
+
+    Args:
+        warning (Warning) : Bokeh warning to check
+
+    Returns:
+        bool
+
+    '''
+    if len(warning) > 0:
+        code = warning[0]
+        return code in __silencers__
+    return False
```

Comment:
I think this can just be:
```suggestion
    return code[0] in __silencers__
```
if warning is not a 4-tuple then something bad has gone wrong. 

The reason it's nice to define this predicate is that I think in  the future we may want to define a proper data structure, or at least a `namedtuple` so defining this predicate now will insulate against those changes.